runtime.md

```diff
@@ -49,12 +49,12 @@ The lifecycle describes the timeline of events that happen from when a container
 1. OCI compliant runtime's `create` command is invoked with a reference to the location of the bundle and a unique identifier.
 2. The container's runtime environment MUST be created according to the configuration in [`config.json`](config.md).
    If the runtime is unable to create the environment specified in the [`config.json`](config.md), it MUST generate an error.
-   While the resources requested in the [`config.json`](config.md) MUST be created, the user-specified code (from [`process`](config.md#process-configuration) MUST NOT be run at this time.
+   While the resources requested in the [`config.json`](config.md) MUST be created, the user-specified code (from [`process configuration`](config.md#process-configuration)) MUST NOT be run at this time.
```

Comment:
I'm ok adding “configuration”, but:

1. I think we the link to the config section discussing the `process` setting is sufficient, so we don't *need* to say “configuration”, and
2. If we add it, “configuration” should go outside the backticks (so: “from the `process` configuration`”).